storage/src/block/mod.rs

```diff
@@ -408,6 +408,10 @@ impl BlockStorage {
         self.dag_sync_block_storage.remove(block_id)
     }
 
+    pub fn delete_all_dag_sync_blocks(&self) -> Result<()> {
```

Comment:
I only see you call this function in test code? Does this function use in sync process?

Reply:
Please see: sync/src/tasks/block_sync_task.rs line 761

Reply:
Does delete_dag_sync_block this function still use in some place?

Reply:
Yes, see the new branch: https://github.com/starcoinorg/starcoin/pull/4120